From 83371f243651e522160b434d0393593049f42d8b Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <Dinh.Nguyen@freescale.com>
Date: Thu, 12 Jun 2008 23:38:01 -0500
Subject: [PATCH] ENGR00080177 Fix MX31-ADS USB IRAM build

Resolving CR #ENGR00078893 made the MX31-ADS build broken for USB
IRAM codeling because the updated code no longer uses IRAM_SIZE and
will use USB_IRAM_SIZE.

Signed-off-by: Dinh Nguyen <Dinh.Nguyen@freescale.com>
---
 include/asm-arm/arch-mxc/mx31.h |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/include/asm-arm/arch-mxc/mx31.h b/include/asm-arm/arch-mxc/mx31.h
index 57bbb78..fe9daf0 100644
--- a/include/asm-arm/arch-mxc/mx31.h
+++ b/include/asm-arm/arch-mxc/mx31.h
@@ -87,8 +87,11 @@
 #define IRAM_BASE_ADDR_VIRT	0xF80C0000
 #define IRAM_SIZE		SZ_16K
 
-#ifdef CONFIG_USB_STATIC_IRAM
 #define USB_IRAM_BASE_ADDR	(IRAM_BASE_ADDR)
+#ifdef CONFIG_USB_STATIC_IRAM
+#define USB_IRAM_SIZE	(2*SZ_8K)
+#else
+#define USB_IRAM_SIZE 0
 #endif
 
 /*
-- 
1.5.4.4

